pkg/volume/util/util.go

```diff
@@ -498,20 +498,20 @@ func MapBlockVolume(
 	volumeMapName string,
 	podUID utypes.UID,
 ) error {
-	// map devicePath to global node path as bind mount
```

Comment:
To avoid potential leakage if kubelet restart in the middle of these functions. I wrote in the commit message:
>Since we use pod path to do reconstruction after kubelet restart, this path should be
mounted first and unmounted last.
>
>Note that the path used by CSI "/plugins/kubernetes.io/csi/volumeDevices/publish/\<pv\>/\<pod uid\>"
may still leak. This path is mounted before MapBlockVolume.